spec/presenters/error_presenter_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
-describe ErrorPresenter do
-
-  let(:claim)           { FactoryBot.build :claim }
-
-  let(:filename)        { File.dirname(__FILE__) + '/data/error_messages.en.yml' }
-  let(:presenter)       { ErrorPresenter.new(claim, filename) }
-
+RSpec.describe ErrorPresenter do
+  let(:claim)     { FactoryBot.build(:claim) }
+  let(:filename)  { File.dirname(__FILE__) + '/data/error_messages.en.yml' }
+  let(:presenter) { ErrorPresenter.new(claim, filename) }
 
   describe 'generate_sequence' do
     context 'base class level fieldnames' do
       context 'fieldname present' do
-        it 'should return the value from the error messages file' do
-          expect(presenter.send(:generate_sequence, 'name')).to eq 50
+        let(:field) { 'name' }
+
+        it 'returns the value from the error messages file' do
+          expect(presenter.send(:generate_sequence, field)).to eq 50
         end
       end
 
       context 'fieldname not present' do
-        it 'should return 99999 ' do
-          expect(presenter.send(:generate_sequence, 'nokey')).to eq 99999
+        let(:field) { 'nokey' }
+
+        # NOTE: no idea what this expected value is!?
+        it 'returns 99999' do
```

Comment:
i think this was to ensure error sequences get a value even if there is no such message in the error_messages.yml

better description for test might be
```
it 'returns default sequence' do
```